addon_service/credentials/models.py

```diff
@@ -98,17 +98,12 @@ def authorized_accounts(self):
                 *filter(
                     bool,
                     [
-                        getattr(self, "authorized_storage_account", None),
-                        getattr(self, "authorized_citation_account", None),
-                        getattr(self, "temporary_authorized_storage_account", None),
-                        getattr(self, "temporary_authorized_citation_account", None),
+                        getattr(self, "authorized_account", None),
+                        getattr(self, "temporary_authorized_account", None),
                     ],
                 )
             ]
-        except (
-            ExternalCredentials.authorized_storage_account.RelatedObjectDoesNotExist
-            or ExternalCredentials.authorized_citation_account.RelatedObjectDoesNotExist
-        ):
+        except ExternalCredentials.authorized_storage_account.RelatedObjectDoesNotExist:
             return None
```

Comment:
this would be nice, but query won't succeed during setting credentials, because model is not saved before this property is accessed, and we'd want to save changes only after everything succeeds.